modules/ossm-remove-cleanup.adoc

```diff
@@ -32,7 +32,7 @@ $ oc delete validatingwebhookconfiguration/openshift-operators.servicemesh-resou
 +
 [source,terminal]
 ----
-$ oc delete mutatingwebhookconfigurations/openshift-operators.servicemesh-resources.maistra.io
+$ oc delete mutatingwebhookconfiguration/openshift-operators.servicemesh-resources.maistra.io
```

Comment:
This typo also appears in the v1 docs.  Normally I'd say those are out of support and not bother with updating them, but since this is the "removing" topic, I'd say it's worth fixing this in the v1 modules (ossm-remove-cleanup-1x).

Reply:
Strictly speaking this is not a typo. Kubernetes accepts both singular and plural forms.